content/en/docs/components/model-registry/overview.md

```diff
@@ -20,9 +20,79 @@ It provides a central interface for all stakeholders in the ML lifecycle to coll
+- **Deploy**: during deployment, the Model Registry provides information of the approved model versions and associated artifacts, ensuring consistency and traceability across deployment environments.
+- **Monitor**: in the monitoring phase, the Model Registry supports ongoing performance monitoring and model drift detection by maintaining a comprehensive record of deployed models and linking to their performance metrics, facilitating proactive maintenance and retraining as needed.
+
 DevOps, Data Scientists, and developers need to collaborate with other users in the ML workflow to get models into production.
 Data scientists need an efficient way to share model versions, artifacts and metadata with other users that need access to those models as part of the MLOps workflow.
 
+## Use Cases
+
+This section describes Model Registry use-cases in the context of a MLOps Platform with Model Training, Experimentation, and Deployment.
+
+A company, ACME Inc., is developing a machine-learning model for predicting customer churn. They require a centralized model registry for their MLOps platform (based on Kubeflow) for managing their ML model development lifecycle, including training, experimentation, and deployment. They want to ensure model governance, reproducibility, and efficient collaboration across data scientists and engineers.
+
+### Persona:
+
+* **Data Scientist**: Responsible for developing and evaluating different models for customer churn prediction. They need to track the performance of various model versions and compare them easily.
+* **MLOps Engineer**: Responsible for deploying the chosen model into production. They need to access the latest model version and its metadata to configure the deployment environment.
+* **Business Analyst**: Monitors the deployed model's performance and makes decisions based on its predictions. They need to understand the lineage of the model and its operational metrics.
+
+### Use Case 1: Tracking the Training of Models
+
+The_ Data Scientist_ uses Kubeflow Notebooks to perform exploratory research and trains several types of models, with different hyperparameters and metrics. The Kubeflow Model Registry is used to track those models, in order to make comparisons and identify the best-performing model. Once the champion model is selected, the _Data Scientist _shares the model with the team. The _Data Scientist_ also tracks the lineage of training data sources and notebook code.
+
+* Track models available on storage: once the model is stored, it can then be tracked in the Kubeflow Model Registry for managing its lifecycle. The Model Registry the registry can catalog, list, index, share, record, organize this information. This allows the _Data Scientist_ to compare different versions and revert to previous versions if needed.
```

Comment:
This seems duplicated.

```suggestion
* Track models available on storage: once the model is stored, it can then be tracked in the Kubeflow Model Registry for managing its lifecycle. The Model Registry can catalog, list, index, share, record, organize this information. This allows the _Data Scientist_ to compare different versions and revert to previous versions if needed.
```

Reply:
oops thanks!

Reply:
oops thanks!